input_definition.go

```diff
@@ -254,6 +265,56 @@ type InputDefinitionInfo struct {
 	Fields []InputDefinitionField `json:"fields"`
 }
 
+// Validate the InputDefinitionInfo data
+func (i *InputDefinitionInfo) Validate(columnLabel string) error {
+	numPrimaryKey := 0
+	accountRowID := make(map[string]uint64)
+
+	if len(i.Frames) == 0 {
+		return fmt.Errorf("At least one frame is required per Input Definition")
+	}
+
+	for _, frame := range i.Frames {
+		if err := frame.Validate(); err != nil {
```

Comment:
if the Validate func (line 246-253) only call ValidateName func, does calling ValidateName(frame.Name) itself here has the same result without adding extra code:
```
if err := ValidateName(frame.Name); err != nil {
			return err
		}
```